keys/cryptostore/generator.go

```diff
@@ -32,12 +36,25 @@ func genSecp256(secret []byte) crypto.PrivKey {
 	return crypto.GenPrivKeySecp256k1FromSecret(secret).Wrap()
 }
 
+// secret is completely ignored for the ledger...
+// just for interface compatibility
+func genLedger(secret []byte) crypto.PrivKey {
+	key, err := nano.NewPrivKeyLedger()
+	if err != nil {
+		// TODO: cleaner error handling
+		panic(err)
+	}
+	return key
+}
+
 func getGenerator(algo string) (Generator, error) {
 	switch algo {
 	case crypto.NameEd25519:
 		return GenEd25519, nil
 	case crypto.NameSecp256k1:
 		return GenSecp256k1, nil
+	case nano.NameLedger:
```

Comment:
IMO this should NameLedgerEd25519

The ledger can support more key types in the future.